### DISCUSSION TITLE
Why my Vercel deployment is not up to date ?

### DISCUSSION THREAD

**Discussion by berk-karademir** [community/community#146934] [Repositories] [answered]
### Select Topic Area

Question

### Body

Why is not getting commit+pushes ? Am i doing something wrong?
For example: this repo is not up to date for 24 hours although many commits.
[my repo](https://github.com/berk-karademir/BekoStore)

**Comment by davevad93** [comment]
Hi @berk-karademir ,

It could be a configuration issue within your `Vercel` environment, log in to [Vercel](https://vercel.com/) with your GitHub account and check in the **Projects** section that your repo is correctly connected.

**Reply by berk-karademir** [reply]
edit: it is now up to date with last commit, i don't know what's happening :)
![ss](https://github.com/user-attachments/assets/593a177f-51b4-4edd-a064-c0b7de5ee4ac)

**Reply by davevad93** [reply, marked as answer]
Hey @berk-karademir , in the `Deployments` section of your `Vercel` project it points out that your latest deplyment is `Commit a95ede9` which is from yesterday.

<img width="500" alt="000" src="https://github.com/user-attachments/assets/f437b918-71bf-4252-8e2d-1b09d6920dc2" />

I noticed that your commits from the last 24 hours don't have any checks, neither green nor red, which could indicate that your git integration inside `Vercel` got broken somehow, that's why it's not updating your deployment.

<img width="383" alt="0" src="https://github.com/user-attachments/assets/03116f7e-edb0-4c6c-9f37-b02dc8693c13" />
<br>
<br>

Try if you can deploy your last commit manually by doing the following: 

1. From the `Vercel` dashboard, select your project.
2. Select the `Deployments` tab. Once on the `Deployments` page, select the `Create Deployments` button in the three dots to the right of the `Deployments` header.

Depending on how you would like to deploy, enter the following:

- Targeted Deployments: Provide the unique ID (SHA) of a commit to build a deployment based on that specific commit.

or

- Branch-Based Deployments: Provide the full name of a branch when you want to build the most recent changes from that specific branch.

If you can't deploy your latest changes even manually, I suggest you delete the project and reconnect your repository to `Vercel`.
